arches/app/media/css/arches.css

```diff
@@ -7703,6 +7703,21 @@ a.mega-dropdown-toggle.disabled {
     padding-left: 15px;
 }
 
+/* .childlessborder {
```

Comment:
@whatisgalen can you please remove this commented out code, as it seems to be cruft